extension/src/experiments/columns/collect/index.test.ts

```diff
@@ -8,9 +8,13 @@ import columnsFixture from '../../../test/fixtures/expShow/columns'
 import workspaceChangesFixture from '../../../test/fixtures/expShow/workspaceChanges'
 import uncommittedDepsFixture from '../../../test/fixtures/expShow/uncommittedDeps'
 import { ExperimentsOutput } from '../../../cli/dvc/reader'
+import { getConfigValue } from '../../../vscode/config'
 
 jest.mock('../../../vscode/config')
 
+const mockedGetConfigValue = jest.mocked(getConfigValue)
+mockedGetConfigValue.mockImplementation(() => 5)
```

Comment:
[I] The usual pattern is to use `beforeEach` with `resetAllMocks` to keep tests independent

```
beforeEach(() => {
  jest.resetAllMocks()
})
```
I thin you could use `mockReturnValueOnce` instead of `mockImplementation` as well. Final code would probably be 

```
beforeEach(() => {
  jest.resetAllMocks()
  mockedGetConfigValue.mockReturnValueOnce(5)
})
```

for this particular file as we are not varying the option.